go/runtime/config/config_test.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+
+	"github.com/oasisprotocol/oasis-core/go/runtime/bundle/component"
+)
+
+func TestComponentConfig(t *testing.T) {
+	require := require.New(t)
+
+	cfg := Config{
+		Components: []ComponentConfig{
+			{
+				ID:       component.ID{Kind: component.ROFL, Name: "foo-test"},
+				Disabled: false,
+			},
+		},
+	}
+
+	compCfg, ok := cfg.GetComponent(component.ID{Kind: component.ROFL, Name: "foo-test"})
+	require.True(ok)
+	require.EqualValues(compCfg.ID.Kind, component.ROFL)
```

Comment:
Any special reason why you didn't use just `Equal`? :thinking:

Reply:
Habit :) In this case it would work as well.